deployments/finance_data_to_s3.yaml

```diff
@@ -0,0 +1,55 @@
+###
+### A complete description of a Prefect Deployment for flow 'Finance Data Publishing'
+###
+name: Finance Data Publishing
+description: null
```

Comment:
Can you add a description of what this does and the path to the finance data repo?